presto-main/src/main/java/com/facebook/presto/execution/SqlQueryManager.java

```diff
@@ -312,4 +323,19 @@ private void enforceCpuLimits()
             }
         }
     }
+
+    /**
+     * Enforce query scan physical bytes limits
+     */
+    private void enforceScanLimits()
+    {
+        for (QueryExecution query : queryTracker.getAllQueries()) {
+            DataSize scan = query.getRawInputDataSize();
```

Comment:
nit - lets rename this as `rawInputSize`

Reply:
fixed, thanks